src/main/webapp/i18n/en/referral.json

```diff
@@ -19,11 +19,15 @@
   },
   "referral": {
     "labels": {
+      "close": "Close",
+      "homePage": "Go to Home Page",
       "or": "OR",
-      "send": "Send"
+      "send": "Send",
+      "sent": "Congratulations, referral sent!"
```

Comment:
this is misleading as we have another tab for sending referrals, I think it should say 'Beneficiary checked in successfully'

Reply:
Fair point, will do.